branching.md

```
@@ -1 +1,3 @@
 #this is the branching stratergy for git
```

Markdownlint

No space after hash on atx style heading

(MD018, no-missing-space-atx)